backend/core/src/listings/listings.service.ts

```diff
@@ -35,21 +37,16 @@ export class ListingsService {
       .leftJoinAndSelect("listings.reservedCommunityType", "reservedCommunityType")
   }
 
-  public async list(
-    origin: string,
-    params: ListingsListQueryParams
-  ): Promise<PaginatedListingsDto> {
-    let query = this.getQueryBuilder().orderBy({
+  public async list(origin: string, params: ListingsQueryParams): Promise<PaginatedListingsDto> {
+    let qb = this.getQueryBuilder()
```

Comment:
Maybe this is a nit, but upstream has `let query = this.getQueryBuilder()`.  If you could submit a rename commit upstream, then we can keep these unified and not have worse merge conflicts.

Reply:
Are you sure? It looks like upstream is `qb`:
https://github.com/bloom-housing/bloom/blob/0b1cace2d75ef3d64a01f67a22185366ee9bc48d/backend/core/src/listings/listings.service.ts#L36

Reply:
You're right! Maybe I was looking at the comparison in the wrong direction or something.